winrm.go

```diff
@@ -359,10 +359,13 @@ func (w *WinRMClient) copyToTempFile(shellId, script string) (string, error) {
 	filename = resp
 	scriptsArray := strings.Split(script, SCRIPTSEPARATOR)
 	for _, scp := range scriptsArray {
+		if len(scp) == 0 {
```

Comment:
strings.TrimSpace(scp), then find the length.

Reply:
> strings.TrimSpace(scp), then find the length.

Done...